src/api/estimatedNextBatchTime.js

```diff
@@ -1,12 +1,12 @@
-const { get } = require('request-promise')
+const get = require('../lib/dvf/get-generic')
 
 module.exports = async dvf => {
   // avoid browser cache with timestamp as querystring
   const t = Date.now()
 
   const url = `${dvf.config.api}/v1/trading/r/estimatedNextBatchTime?t=${t}`
   try {
-    const data = await get(url)
+    const data = await get(dvf, url)
     return JSON.parse(data)
   }
   catch(e) {
```

Comment:
the catch here is unnecessary, so pulling all the comment together the function should become:

```js
module.exports = async dvf => {
  // avoid browser cache with timestamp as querystring
  const t = Date.now()
  const url = `/v1/trading/r/estimatedNextBatchTime?t=${t}`
  return get(dvf, url)
}
```